### PR TITLE
[OptApp] Restructure 10 - Add control transformation technique

### PR DESCRIPTION
**Pull request by sunethwarna**
**📝 Description**
This PR adds a new class named `ControlTranformationTechnique`. It is again uses the Bridge pattern to work with `Control` and `TransformationTechniues` in `OptimizationApplication`

Todo:
- [ ] Add test

**🆕 Changelog**
- Add `ControlTransformationTechnque`

### CODE REVIEW

File: applications/OptimizationApplication/python_scripts/utilities/control_transformation_technique.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    def TransformSensitivity(self, collective_container_variable_data_holder: KratosOA.CollectiveVariableDataHolder):
+        for container in collective_container_variable_data_holder.GetVariableDataHolders():
+            CallOnAll(self.__transformation_techniques, TransformationTechnique.TransformSensitivity, container)
+
+    def TransformUpdate(self, collective_container_variable_data_holder: KratosOA.CollectiveVariableDataHolder):
+        for container in collective_container_variable_data_holder.GetVariableDataHolders():
+            CallOnAll(self.__transformation_techniques, TransformationTechnique.TransformUpdate, container)
+
+    def SetControlUpdate(self, collective_container_variable_data_holder: KratosOA.CollectiveVariableDataHolder):
+        WriteCollectiveVariableDataHolderToOptmizationInfo(
```
Comment (FMeister) [inline applications/OptimizationApplication/python_scripts/utilities/control_transformation_technique.py:57]:
```suggestion
        WriteCollectiveVariableDataHolderToOptimizationInfo(
```
Typo

File: applications/OptimizationApplication/python_scripts/utilities/control_transformation_technique.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            self.__optimization_info,
```
Comment (FMeister) [inline applications/OptimizationApplication/python_scripts/utilities/control_transformation_technique.py:58]: Change function argument order to match the name of the function.

**Review by FMeister — COMMENTED** [review]
Just some small comments from my side.

**Comment by sunethwarna** [comment]
Closing since this is no longer required.

**Pull request closed without merging**